docs/user-guide/ingest-data/for-observability/opentelemetry-collector.md

```diff
@@ -0,0 +1,114 @@
+---
+keywords: [OpenTelemetry Collector, OpenTelemetry, traces, data pipeline]
+description: Introduction to using OpenTelemetry Collector to write traces data to GreptimeDB.
+---
+
+# OpenTelemetry Traces
+
+## Overview
+
+[OpenTelemetry Collector](https://opentelemetry.io/docs/collector/) is a vendor-neutral service for collecting and processing OpenTelemetry data. In this article, we will introduce how to use OpenTelemetry Collector to write traces data to GreptimeDB.
+
+## Quick Start
+
+:::note
+Before starting the OpenTelemetry Collector, please ensure that GreptimeDB is running.
+:::
+
+### Start OpenTelemetry Collector
+
+You can use the following command to quickly start an OpenTelemetry Collector instance, which will listen on ports `4317` (gRPC) and `4318` (HTTP):
+
+```shell
+docker run --rm \
+  --network host \
+  -p 4317:4317 \
+  -p 4318:4318 \
+  -v $(pwd)/config.yaml:/etc/otelcol-contrib/config.yaml \
+  otel/opentelemetry-collector-contrib:0.123.0
+```
+
+The content of the `config.yaml` file is as follows:
+
+```yaml
+receivers:
+  otlp:
+    protocols:
+      grpc:
+        endpoint: 0.0.0.0:4317
+      http:
+        endpoint: 0.0.0.0:4318
+
+exporters:
+  otlphttp:
```

Comment:
consider adding "trace" to this name

Reply:
i found out it can't add a custom name.

Reply:
I see. We may have a conflict on pipeline name header when we want to use collector for both traces and logs.